<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hangar</class>
 <widget class="QMainWindow" name="Hangar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="UCM_check">
          <property name="text">
           <string>UCM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Scourge_check">
          <property name="text">
           <string>Scourge</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="PHR_check">
          <property name="text">
           <string>PHR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Resistance_check">
          <property name="text">
           <string>Resistance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="shaltari_check">
          <property name="text">
           <string>Shaltari</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="bomber_radio">
            <property name="text">
             <string>Bomber</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="torpedo_radio">
            <property name="text">
             <string>Torpedo</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="drop_radio">
            <property name="text">
             <string>Drop</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="fighter_radio">
            <property name="text">
             <string>Fighter</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="name_edit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Thrust</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="thrust_spin"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>PD+</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="PD_spin"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Lock</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="lock_combo"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Attack</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="attack_spin"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Damage</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="damage_spin"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="specialAdd_button">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="specialRemove_button">
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="special_combo"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="specialLayout"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="saveWeapon_button">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadWeapon_button">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteWeapon_button">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="launchLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1303</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuSave"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
